runtime/jcl/common/sun_reflect_ConstantPool.c

```diff
@@ -25,6 +25,9 @@
 #include "jclprots.h"
 #include "jclglob.h"
 #include "jniidcacheinit.h"
+#include "j9.h"
+
+#define J9CLASS_FROMJ9CLASSREF(vmThread, j9ClassRef) J9VMJAVALANGINTERNALRAMCLASS_VMREF(vmThread, J9_JNI_UNWRAP_REFERENCE(j9ClassRef))
```

Comment:
`CLASSREF` has classfile constant pool connotations.  This should mention either the ConstantPoolOOP or InternalRamClassRef in the name of the macro so it's clearer what's happening here.